color/src/colorspace.rs

```diff
@@ -283,6 +283,122 @@ impl ColorSpace for DisplayP3 {
+/// gamut over sRGB on that component. It was developed to encompass typical color print gamuts.
+///
+/// Its components are `[r, g, b]` (red, green, and blue channels respectively), with `[0, 0, 0]`
+/// pure black and `[1, 1, 1]` white. The natural bounds of the channels are `[0, 1]`.
+///
+/// This corresponds to the color space in [CSS Color Module Level 4 § 10.5][css-sec] and is
+/// [characterized by the ICC][icc]. Adobe RGB is described [here][adobe] by Adobe.
+///
+/// [css-sec]: https://www.w3.org/TR/css-color-4/#predefined-a98-rgb
+/// [icc]: https://www.color.org/chardata/rgb/adobergb.xalter
+/// [adobe]: https://www.adobe.com/digitalimag/adobergb.html
+#[derive(Clone, Copy, Debug)]
+pub struct A98Rgb;
+
+impl ColorSpace for A98Rgb {
+    const TAG: Option<ColorSpaceTag> = Some(ColorSpaceTag::A98Rgb);
+
+    fn to_linear_srgb([r, g, b]: [f32; 3]) -> [f32; 3] {
+        // XYZ_to_lin_sRGB * lin_A98_to_XYZ
+        #[expect(
+            clippy::cast_possible_truncation,
+            reason = "exact rational, truncate at compile-time"
+        )]
+        const LINEAR_A98RGB_TO_SRGB: [[f32; 3]; 3] = [
+            [
+                (66_942_405. / 47_872_228.) as f32,
+                (-19_070_177. / 47_872_228.) as f32,
+                0.,
+            ],
+            [0., 1., 0.],
+            [
+                0.,
+                (-11_512_411. / 268_173_353.) as f32,
+                (279_685_764. / 268_173_353.) as f32,
+            ],
+        ];
+        matmul(
+            &LINEAR_A98RGB_TO_SRGB,
+            [r, g, b].map(|x| x.abs().powf(2.199_218_8).copysign(x)),
+        )
+    }
+
+    fn from_linear_srgb([r, g, b]: [f32; 3]) -> [f32; 3] {
+        // XYZ_to_lin_A98RGB * lin_sRGB_to_XYZ
+        #[expect(
+            clippy::cast_possible_truncation,
+            reason = "exact rational, truncate at compile-time"
+        )]
+        const LINEAR_SRGB_TO_A98RGB: [[f32; 3]; 3] = [
+            [
+                (47_872_228. / 66_942_405.) as f32,
+                (19_070_177. / 66_942_405.) as f32,
+                0.0,
+            ],
+            [0., 1., 0.],
+            [
+                0.,
+                (11_512_411. / 279_685_764.) as f32,
+                (268_173_353. / 279_685_764.) as f32,
+            ],
+        ];
+        matmul(&LINEAR_SRGB_TO_A98RGB, [r, g, b])
+            .map(|x| x.abs().powf(1. / 2.199_218_8).copysign(x))
+    }
+
+    fn clip([r, g, b]: [f32; 3]) -> [f32; 3] {
+        [r.clamp(0., 1.), g.clamp(0., 1.), b.clamp(0., 1.)]
+    }
+}
+
+/// 🌌 The CIE XYZ color space with a 2° observer and a reference white of D50.
+///
+/// Its components are `[X, Y, Z]`. The components are unbounded, but are usually positive.
+/// Reference white has a luminance `Y` of 1.
+///
+/// This corresponds to the color space in [CSS Color Module Level 4 § 10.8][css-sec]. It is
+/// defined in CIE 015:2018.
+///
```

Comment:
I suggest adding this:

Following the CSS Color 4 spec, the conversion between D50 and D65 white points is done with the standard Bradford linear chromatic adaptation transform.